crates/wasm-mutate/src/mutators/peephole/eggsy/lang.rs

```diff
@@ -700,6 +704,10 @@ lang! {
         F32(u32) = "f32.const",
         /// F64 constant node
         F64(u64) = "f64.const",
+        /// constant ref.null funcref node
+        FuncRefNull = "funcref.null",
+        /// constant ref.null externref node
+        ExternRefNull = "externref.null",
```

Comment:
Oh I appear to have forgotten `ref.null` in the language definition here when I added bulk-memory support. Would you be up for switching this to `RefNull(ValType)` or similar so this can be `ref.null func` and `ref.null extern` (or similar)

Reply:
`ValType` doesn't really fit here all that well – `ValType` would need a `Display` and a `FromStr` implementation, and also it covers more distinct types than `ref.null` can handle either way.

I went ahead and added a `RefType` enum instead with a conversion to `ValType`.